tests/integration_test.py

```diff
@@ -74,8 +76,15 @@ def setup_module():
     try:
         c.inspect_image(BUSYBOX)
     except NotFound:
-        c.pull(BUSYBOX)
-        c.inspect_image(BUSYBOX)
+        os.write(2, "\npulling busybox\n".encode('utf-8'))
+        for data in c.pull('busybox', stream=True):
+            data = json.loads(data.decode('utf-8'))
+            os.write(2, ("%c[2K\r" % 27).encode('utf-8'))
+            status = data.get("status")
+            progress = data.get("progress")
+            detail = "{0} - {1}".format(status, progress).encode('utf-8')
+            os.write(2, detail)
+        os.write(2, "\npulled busybox\n".encode('utf-8'))
```

Comment:
I believe @mnowster meant adding the line `c.inspect_image(BUSBOX)` at the end here so if there's been an issue during the pull, it will be detected immediately, making it easier to pinpoint why the tests have failed if it occurred. Having the pull logs in addition to that is fine as far as I'm concerned :+1: